book/chrome.md

```diff
@@ -912,7 +913,7 @@ class Browser:
 
 Note that this display list is always drawn at the top of the window,
 unlike the tab contents (which scroll). Make sure to draw the chrome
-*after* the main tab contents, so that the chrome ends up on top.
+*after* the main tab contents, so that the chrome is drawn over it.
```

Comment:
We mean "top" in terms of stacking but it reads like it could mean 2D layout.

Reply:
not critical IMO